docker/Makefile

```diff
@@ -14,7 +14,15 @@ image: prepare
 	docker build -f Dockerfile -t $(IMAGE_NAME) $(BUILD_DIR)
 
 run: image
-	docker-compose -f docker-compose.yml up
+    # This will not work on Windows
+	@if [ -f .env ]; then\
+		docker-compose -f docker-compose.yml --env-file .env up;\
+	else\
+		docker-compose -f docker-compose.yml up;\
+	fi
+
+run-postgres:
+	docker-compose -f docker-compose.yml up postgres
```

Comment:
Would it make sense to add `-d` here so it runs in the background?

Reply:
Personally I like to have it running in the foreground so I can easily see the logs and shut it down. I can add something like `make background=true run-postgres` if you'd like.

Reply:
Nah, no need for that. This is fine.